src/java/main/org/apache/zookeeper/server/quorum/QuorumPeerConfig.java

```diff
@@ -61,6 +61,7 @@
 
     private static boolean standaloneEnabled = true;
     private static boolean reconfigEnabled = false;
+    private static int requestWarnThresholdMs = 10000;
```

Comment:
Is 2 or 3 seconds reasonable? I have seen 2.3 seconds as max latency sometimes, however I don't have much experience.